src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/SubscriptionAT.java

```diff
@@ -117,6 +110,59 @@ public void testOffsetsCommit() throws Exception {
         assertThat(committedOffset, equalTo("25"));
     }
 
+    @Test
+    public void testGetSubscriptionCursors() throws IOException {
+        final EventType eventType = createEventType();
+        final Subscription subscription = createSubscription(eventType);
+        commitCursors(subscription);
+
+        List<Cursor> actualCursors = getSubscriptionCursors(subscription);
+        Assert.assertEquals(Arrays.asList(new Cursor("0", "25")), actualCursors);
+    }
+
+    @Test
+    public void testGetSubscriptionCursorsEmpty() throws IOException {
+        final EventType eventType = createEventType();
+        final Subscription subscription = createSubscription(eventType);
+
+        Assert.assertTrue(getSubscriptionCursors(subscription).isEmpty());
+    }
+
+    @Test
+    public void testGetSubscriptionNotFound() throws IOException {
+        final EventType eventType = createEventType();
+        createSubscription(eventType);
+
+        given()
+                .get(format(CURSORS_URL, "UNKNOWN_SUB_ID"))
+                .then()
+                .statusCode(HttpStatus.SC_NOT_FOUND);
+    }
+
+    private Subscription createSubscription(EventType eventType) throws IOException {
+        final String subscriptionJson = "{\"owning_application\":\"app\",\"event_types\":[\"" + eventType.getName() + "\"]}";
+        final Response response = given()
+                .body(subscriptionJson)
+                .contentType(JSON)
+                .post(SUBSCRIPTIONS_URL);
+        return MAPPER.readValue(response.print(), Subscription.class);
+    }
+
+    private void commitCursors(Subscription subscription) {
```

Comment:
I think this method should take cursors as a second parameter (doesn't really matter in which form: json string or objects)
Because currently the method looks a bit strange as it commits very specific cursors.